src/shared/services/map-location/index.js

```diff
@@ -1,28 +1,114 @@
-function mapLocation(loc) {
+export const location2feature = location => ({
+  type: 'Point',
+  coordinates: [location.lng, location.lat],
+});
+
+export const feature2location = feature => {
+  const { coordinates } = feature;
+  return {
+    lat: coordinates[1],
+    lng: coordinates[0],
+  };
+};
+
+/**
+ * converts the sia location in latlon fromat
```

Comment:
Just the 'sia' location or any location?
fromat should be format

Reply:
yes, the location is the geometry and addres.